toolkit/tools/pkg/imagecustomizerlib/createuuid.go

```diff
@@ -47,3 +51,61 @@ func convertUuidToString(uuid [UuidSize]byte) string {
 
 	return uuidStr
 }
+
+func extractImageUUID(imageConnection *ImageConnection) ([UuidSize]byte, string, error) {
+	var emptyUuid [UuidSize]byte
+
+	releasePath := filepath.Join(imageConnection.Chroot().RootDir(), "etc/image-customizer-release")
```

Comment:
It would be good to move `"etc/image-customizer-release"` into a shared constant.

Reply:
Fixed.